docs/running-rules.md

```diff
@@ -28,9 +28,14 @@ You can run all rules on your code locally or continuously in your Source Code M
 
 ## Running Semgrep Registry rules locally
 
-You can test Semgrep Registry rules with the following command:
-<pre class="language-bash"><code>semgrep --config=auto <span className="placeholder">PATH/TO/SRC</span></code></pre>
-Substitute the optional placeholder <code><span className="placeholder">PATH/TO/SRC</span></code> with the path to your source code.
+You can scan your current working directory with pre-selected Semgrep Registry rules with the following command:
```

Comment:
@milanwilliams It occurred to me that the "spirit" of this running-rules guide was to run rules in many ways, so it seemed weird to change some of the commands to semgrep ci (especially because subsequent guides were variations on the base semgrep scan). 

I included semgrep login && semgrep ci as an alternate path instead. Let me know if this is ok.

Reply:
I think this is good for now 👍 Though I'd reverse the order so `semgrep login && semgrep ci` is the first option in this list, so it's clearer it's the preferred route :)

Reply:
I did indeed reverse the order :)